var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -234,3 +234,31 @@
     line: '{{ item.key }}="{{ item.value }}"'
   with_items:
     - { key: 'JAVA_HOME', value: '/usr/lib/jvm/oracle-java8-jdk-amd64' }
+
+- shell: dpkg-query -Wf '${Package}' 'delphix-platform-*' | sed 's/delphix-platform-//'
+  register: determine_platform
+
+# Configure the Azure agent.
+- lineinfile:
+    dest: /etc/waagent.conf
+    regexp: '^{{ item.key }}='
+    line: '{{ item.key }}={{ item.value }}'
+  with_items:
+    #
+    # We use cloud-init rather than the Azure agent to handle any
+    # provisioning logic that we need.
+    #
+    - { key: 'Provisioning.Enabled', value: 'n' }
```

Comment:
Do we need to do anything to prevent users from being added to the delphix engine?

Reply:
Creating a new user is done as part of provisioning, so I think disabling provisioning should be sufficient. I am planning on asking for access to the Azure console so that I can confirm this though.